### PR TITLE
add QE dedicated azure stack cluster profile

### PR DESCRIPTION
**Pull request by jinyunma**
along with https://github.com/openshift/ci-tools/pull/3177

### CODE REVIEW

**Comment by jinyunma** [comment]
@jianlinliu PTAL, thanks.

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 1]
```diff
@@ -1939,6 +1939,36 @@ secret_configs:
...
+        registry_url: quay.io
+      - auth_field: auth
+        email_field: email
+        item: registry.connect.redhat.com-pull-secret
+        registry_url: registry.connect.redhat.com
+      - auth_field: auth
+        email_field: email
+        item: registry.redhat.io-pull-secret
+        registry_url: registry.redhat.io
+      - auth_field: token_image-puller_build02_reg_auth_value.txt
```
Comment (jianlinliu) [inline core-services/ci-secret-bootstrap/_config.yaml:1964]: Is this a MUST?

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 2]
```diff
@@ -1939,6 +1939,36 @@ secret_configs:
...
+        item: build_farm
+        registry_url: registry.build02.ci.openshift.org
+  to:
+  - cluster_groups:
```
Comment (jianlinliu) [inline core-services/ci-secret-bootstrap/_config.yaml:1968]: Sounds like we can we append the array item to https://github.com/openshift/release/blob/923a6713e8ba3c4c60bbce2b51ce3d816248cbf1/core-services/ci-secret-bootstrap/_config.yaml#L1897-L1905

Comment (jinyunma) [reply core-services/ci-secret-bootstrap/_config.yaml:1968]: I copied the content from here. okay, will update to append here.

**Comment by bear-redhat** [comment]
/approve
/hold
Feel free to cancel the hold once https://github.com/openshift/ci-tools/pull/3177 is merged

**Pull request merged**